src/DataWrangling/metadata.jl

```diff
@@ -205,13 +205,6 @@ end
 #### Metadata interface
 ####
 
-"""
-    download_dataset(metadata)
-
-Download the dataset file(s) specified by `metadata` and return the path(s) to the downloaded file.
-"""
-function download_dataset end
-
```

Comment:
why this got deleted? don't all the submodules import it and add a method?

Reply:
This is a duplicate definition, so unneeded